build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
   id 'pmd'
```

Comment:
The data-models jar needs to be released into jitpack to be used by other projects later. So the lib version should be upgraded and the new version needs to be used in other projects